app/controllers/planning_application/site_notices_controller.rb

```diff
@@ -101,7 +106,9 @@ def send_mail
       if site_notice_params[:internal_team_email].presence
         @planning_application.send_internal_team_site_notice_mail(site_notice_params[:internal_team_email])
       else
-        @planning_application.send_site_notice_mail(@planning_application.applicant_email)
+        @planning_application.send_site_notice_mail(
+          @planning_application.applicant_email || @planning_application.agent_email
```

Comment:
I'm not completely sure here but should the agent email come first? Use the agent if one exists otherwise use the applicant.

If having an agent always means there's no applicant email then this doesn't matter anyway.